mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlConnector.java

```diff
@@ -0,0 +1,77 @@
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ *  MySQL connector
+ */
+public class MysqlConnector {
```

Comment:
Can we rename it to `MysqlClient` or `MysqlHelper`? 
Want to avoid conflict with CDAP connector terminology.